packages/@uppy/tus/src/index.js

```diff
@@ -23,18 +23,27 @@ const getFingerprint = require('./getFingerprint')
  */
 const tusDefaultOptions = {
   endpoint: '',
-  resume: true,
+
```

Comment:
Pulled in the latest default options. Two notable changes:
- the resume option has been removed (see comments below)
- the retryDelays option is enabled by default